### PR TITLE
continue of PR #740

### PR DESCRIPTION
**Pull request by aneox**
- Added Job order combobox to job processor wizard
- Added checkbox to strip feeders, to use different approach of rotation https://github.com/openpnp/openpnp/issues/715

### CODE REVIEW

File: src/main/java/org/openpnp/machine/reference/ReferencePnpJobProcessor.java [hunk 1]
```diff
@@ -426,10 +432,26 @@ protected void doPlan() throws Exception {
...
+            jobPlacements = getPendingJobPlacements().stream()
+                    .sorted(Comparator.comparing(JobPlacement::getPartHeight))
+                    .collect(Collectors.toList());
+        } else {
+            if (this.jobOrder == ORDER_JOB_PART) {
+                // Get the list of unfinished placements and sort them by part.
+                jobPlacements = getPendingJobPlacements().stream()
+                        .sorted(Comparator.comparing(JobPlacement::getPartId))
+                        .collect(Collectors.toList());
+            } else {
```
Comment (vonnieda) [inline src/main/java/org/openpnp/machine/reference/ReferencePnpJobProcessor.java]: This seems to be a duplicate of the above code?

File: src/main/java/org/openpnp/spi/Machine.java [hunk 2]
```diff
@@ -163,6 +163,10 @@
 
     public boolean getHomeAfterEnabled();
 
+    public boolean getUsePickRotationInsteadOfRotationInTapeForStripFeeders();
```
Comment (vonnieda) [inline src/main/java/org/openpnp/spi/Machine.java]: As I said before, this does not belong on Machine. The property should be on the strip feeder it belongs to. If you want a global default, you can use machine.setProperty() and machine.getProperty() from the ReferenceStripFeederWizard.

File: src/main/java/org/openpnp/machine/reference/ReferencePnpJobProcessor.java (hunk 1 above)
Comment (aneox) [reply src/main/java/org/openpnp/machine/reference/ReferencePnpJobProcessor.java]: it is like default value if string is wrong

Comment (aneox) [reply src/main/java/org/openpnp/machine/reference/ReferencePnpJobProcessor.java]: anyone can easy expand list of sort directions

File: src/main/java/org/openpnp/spi/Machine.java (hunk 2 above)
Comment (aneox) [reply src/main/java/org/openpnp/spi/Machine.java]: ok, i didnt know about machine.setProperty() and machine.getProperty(), will fix it

Comment (Phara0h) [reply src/main/java/org/openpnp/spi/Machine.java]: Can this be renamed to something that is not a sentence? like "isPickRotation()"

**Comment by aneox** [comment]
Hi @vonnieda !
Is this fix correct?

**Comment by vonnieda** [comment]
* Merged the job order checkbox.
* Rejected strip feeder rotation change. See https://github.com/openpnp/openpnp/issues/715#issuecomment-441534350.

**Pull request merged**